schemas/process.yml

```diff
@@ -202,6 +202,28 @@
       description: >
         Thread name.
 
+    - name: thread.capabilities.permitted
+      level: extended
+      type: keyword
+      short: Array of capabilities a thread could assume.
+      description: >
+        This is a limiting superset for the effective capabilities that the
+        thread may assume.
+      example: "[\"CAP_BPF\", \"CAP_SYS_ADMIN\"]"
```

Comment:
If this is the format we expect to use then I recommend adding a `pattern` so that automated validation can yield warnings when the format is not followed. I am thinking `^(CAP_[A-Z_]+|\d+)$` which allows software to insert the capability number if it is unaware the associated name.

example: https://github.com/elastic/ecs/blob/ce9dbe7f9b1f276356d271a22b66c72b5acd8aa7/schemas/source.yml#L70

Reply:
02188a6